controllers/route.go

```diff
@@ -234,6 +235,13 @@ func (as *AdminServer) UserManagement(w http.ResponseWriter, r *http.Request) {
 	getTemplate(w, "users").ExecuteTemplate(w, "base", params)
 }
 
+// Webhook is an admin-only handler that handles webhooks
```

Comment:
In this case, your comment starts with `Webhook ...`, while it should be `Webhooks` - the name of the function. `golint` can help you track down these errors.

Reply:
fixed